### PR TITLE
Japanese translation

### PR DESCRIPTION
**Pull request by yuki-gh**
not tested yet because local build doesn't read lang files

### CODE REVIEW

File: WhyNotWin11.au3 [hunk 1]
```diff
@@ -213,7 +213,7 @@ Func Main()
 	GUICtrlSetBkColor(-1, _HighContrast(0xF2F2F2))
 
 	If Not @MUILang = 0409 Then
-		GUICtrlCreateLabel(("Translation by") & " " & _GetTranslationCredit(), 130, 570, 250, 20, $SS_CENTERIMAGE)
+		GUICtrlCreateLabel(_Translate("Translation by") & " " & _GetTranslationCredit(), 130, 570, 250, 20, $SS_CENTERIMAGE)
```
Comment (micwoj92) [inline WhyNotWin11.au3:216]: Why did you add that?

Comment (yuki-gh) [reply WhyNotWin11.au3:216]: Because I thought it should be read from lang file.
